views/forms/index.ejs

```diff
@@ -1,18 +1,20 @@
 <%- contentFor('title') %>Form
 <%- contentFor('body') %>
-<h1>Forms Page</h1>
+<h1>AB-540 Application</h1>
 
-
-<a href="/forms/new">Add a New Forms</a>
-<p>Forms for Undocument Immigrants</p>
-<ul>
+<b>What is a AB-540?</b><br>
+<a>AB-540 allows undocmented people to pay instate tuition in California instead of paying a out of state fees.</a>
+ <p align="right"><a href="https://ab540.com/What_Is_AB540_.html">show more!>></a></p>
+<b>AB-540 Applications</b><br>
+<b>Find your school below:</b>
+<ul align="center">
   <% for (let record of records) { %>
     <li>
-      <a href="/forms/<%= record.id %>">
-        <% if (record.logo) { %>
-          <img src="<%= assetUrl(record.logo) %>" />
-        <% } %>
-        <%= record.name %> <%= record.year %><br><%= record.school %><br><%= record.url %><br><%= record.lang %></a><br>
-      </li>
+      <a href="/forms/<%= record.id %>"><%= record.name %> <%= record.year %></a><p></p>
+      <% if (record.logo) { %>
+        <img src="<%= assetUrl(record.logo) %>" /><p></p>
+      <% } %>
+      <a style=border-radius:50px; type="button" class="btn btn-primary" href="<%= record.url %>">AB-540 for <%= record.school %></a><p></p>
```

Comment:
Leroy, I see you've added the border-radius style here to make the rounded buttons.

However, looking at the Figma designs, these rounded buttons are going to be used in a lot of pages. So, I'd recommend adding this style to the button class in a css stylesheet linked to the main layout page.